tutorials/persistance-comparison-series/File/file.md

```diff
@@ -0,0 +1,345 @@
+# Saving Data in Unity3D using Files
+<b>(Part 2 of the Persistence Comparison Series)</b>
+
+## Introduction
+
+Persisting data is an important part of most games. Unity offers only a limited set of solutions which means we have to look around for other options as well.
+
+In Part 1 of this series we explored Unity's own solution: `PlayerPrefs`. This time we look into one of the ways we can use the underlying .NET framework by saving files. Here is an overview of the complete series:
+
+- Part 1: [PlayerPrefs](https://github.com/realm/unity-examples/blob/persistence-comparison/tutorials/persistance-comparison-series/PlayerPrefs/player_prefs.md)
+- Part 2: Files *(this tutorial)*
+- Part 3: BinaryReader and BinaryWriter *(coming soon)*
+- Part 4: SQL
+- Part 5: Realm Unity SDK
+- Part 6: Comparison of all those options
+
+Like part 1 this tutorial can also be found in the https://github.com/realm/unity-examples repository on the [persistence-comparison](https://github.com/realm/unity-examples/tree/persistence-comparison) branch.
```

Comment:
Should the branch be https://github.com/realm/unity-examples/tree/df/persistence-comparison-file or the plan is to merge everything in https://github.com/realm/unity-examples/tree/df/persistence-comparison?

Reply:
The plan is to merge them all into `persistent-comparison` so that the whole series lives in one branch and is easy accessible. Also, since the same game is used, it's kinda nice to see them all work side by side and compare them.